ansible/roles/aws_ecs/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
     - build_ami
 - shell: /usr/sbin/make_dockercfg {{docker_user|quote}} {{docker_pass|quote}} {{docker_email|quote}} {{docker_registry|quote}}
   register: dockercfg
+- copy: 'content="{{dockercfg.stdout}}" dest=/root/.dockercfg owner=root group=root mode=0600'
```

Comment:
You shouldn't need the single quotes around this I don't believe?  We don't use it anywhere else.